doc/Getting_Started.md

```diff
@@ -61,6 +63,7 @@ $ pip3 install --upgrade tockloader
 
 Ubuntu:
 ```
+$ apt install -y build-essential gcc-arm-none-eabi python3-pip curl
```

Comment:
Why do we need `gcc-arm-none-eabi`?

Reply:
That should be resolved in the updated commit.

Reply:
But even when building for ARM based boards gcc-arm-none-eabi shouldn't be needed.